ambry-protocol/src/main/java/com.github.ambry.protocol/MessageInfoListSerde.java

```diff
@@ -78,7 +85,7 @@ void serializeMessageInfoList(ByteBuffer outputBuffer) {
         outputBuffer.putLong(messageInfo.getSize());
```

Comment:
Just feel the name of this method is a big confusing, as not clear if getting the size of the messageInfo itself, or getting the size of the blob. Not a big deal, and not caused by this patch.